storybook/storybook-react/src/components/Blockquote/Blockquote.stories.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Blockquote } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Blockquote',
+  component: Blockquote,
+  args: {
+    children: 'Jouw typograaf biedt mij zulke exquise schreven!',
+    inverseColor: false,
+  },
+  argTypes: {
+    inverseColor: { control: 'boolean' },
+  },
+} satisfies Meta<typeof Blockquote>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  decorators: [
+    (Story, context) => (
+      <div
```

Comment:
We have a couple of these now, can we make it an `.amsterdam-docs-dark-background`?